src/components/node/index.tsx

```diff
@@ -15,14 +15,18 @@ interface NodeProps {
 export const NodePageDisplay = ({node, ...props}: NodeProps) => {
   return (
     <>
-      {node.__typename === "NodeStanfordCourse" && <StanfordCourse node={node} {...props} />}
-      {node.__typename === "NodeStanfordEvent" && <StanfordEvent node={node} {...props} />}
-      {node.__typename === "NodeStanfordEventSeries" && <StanfordEventSeries node={node} {...props} />}
-      {node.__typename === "NodeStanfordNews" && <StanfordNews node={node} {...props} />}
+      {node.__typename === "NodeStanfordCourse" && <StanfordCourse node={node} aria-labelledby={node.id} {...props} />}
+      {node.__typename === "NodeStanfordEvent" && <StanfordEvent node={node} aria-labelledby={node.id} {...props} />}
+      {node.__typename === "NodeStanfordEventSeries" && (
+        <StanfordEventSeries node={node} aria-labelledby={node.id} {...props} />
+      )}
+      {node.__typename === "NodeStanfordNews" && <StanfordNews node={node} aria-labelledby={node.id} {...props} />}
       {node.__typename === "NodeStanfordPage" && <StanfordPage node={node} {...props} />}
```

Comment:
why not here?

Reply:
@pookmish passing `aria-labelledby` into StanfordPage component here is causing the SiteImprove error:
```Issue: ARIA attribute unsupported or prohibited```
This error only appears on basic page content types since their `aria-labelledby` is defined in the parent `<article>` from https://github.com/SU-SWS/sulgryphon-nextjs/blob/1.x/app/%5B...slug%5D/page.tsx#L40

![image](https://github.com/user-attachments/assets/66d18b00-35f1-4091-bafa-bb14fbe2a120)

Reply:
Annnd now I just realized we could just change the div to an article instead since [articles can be nested within articles](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/article)? 🤔

Reply:
That should do it 👍 Thank you for double checking this, @pookmish!

![image](https://github.com/user-attachments/assets/9f5c8988-9bcf-457c-ada1-c93e6b1759ab)